requirements.txt

```diff
@@ -13,3 +13,4 @@ fastparquet==0.8.3
 xgboost==1.6.2
 git+https://github.com/aicoe-aiops/ocp-ci-analysis.git#egg=src
 ipynb==0.5.1
+PyGithub
```

Comment:
I think we should specify a version here for PyGithub

Reply:
Version specified.